### ISSUE TITLE
Have a nice process for collecting exceptions when desktop app encounters errors

### ISSUE THREAD

**Issue by timabbott** [zulip/zulip-electron#40]
We should have a nice documented process for when a user experiences something weird in the app, for them to submit any tracebacks that might have happened (even if they launched the app not from a terminal).  Probably this means we want to log the output from the app to a file somewhere reasonable on each platform, so we can tell users "can you send us foo.log"?

**Comment by akashnimare** [comment]
[CrashReporter](https://github.com/electron/electron/blob/master/docs/api/crash-reporter.md) is what we need.

**Comment by timabbott** [comment]
Just to add some context on this, the Zulip webapp already has a javascript error reporter framework called `blueslip` that probably works for any JS errors that happen in the actual Zulip webapp code; we probably still want something for electron tracebacks though.

**Comment by akashnimare** [comment]
Progress is going on in [crash-reporter](https://github.com/zulip/zulip-electron/tree/crash-report) branch.

**Comment by timabbott** [comment]
@akashnimare this is now done, right?

**Comment by akashnimare** [comment]
We need to add few more things in this like an option to reveal logs directory so that user can easily send it to us in case we need it. Also, we need to improve the logging process. Currently it only logs about the autoupdates stuff. We need to add those console errors in the logs as well.

**Comment by abhigyank** [comment]
Since we have added sentry now for reporting errors (for expected and unexpected exceptions), implemented logger-util in the codebase for logging errors locally and also have an option of zipping all the logs so that user can send it to us easily, I think we can mark this issue as solved and close it.

**Comment by akashnimare** [comment]
Correct, we have already implemented all the necessary logging things.
